### ISSUE TITLE
Installing LendingClub in Python 3.8

### ISSUE THREAD

**Issue by Coding4CodingFun** [jgillick/LendingClub#29]
I am unable to install the LendingClub module.  I was on an older version of Python on Windows 10.  I upgraded to Python 3.8.  I upgraded pip.  I still get this error by using

pip install lendingclub or pip3 install lendingclub

ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command 

I'm new to coding and Python and muddling my way through :)

**Comment by anthonymorast** [comment]
I know it's a little late but for anyone else wondering, it looks like this library was built for Python 2.x based on the setup data in setup.py

For me, the install is failing when trying to install pybars. In Python 3 I think the real library is pybars3. Not sure if that helps anyone.

I'm very interested in using this myself and plan on trying to port to python 3.x if not done already.

**Comment by jgillick** [comment]
Yes, this library was written for Python2. I am also no longer actively maintaining this library. If anyone wants to take an attempt at upgrading it to Python3, I will gladly merge the pull request.
